### ISSUE TITLE
`pagelet_resource': undefined method `<<' for nil:NilClass (NoMethodError)

### ISSUE THREAD

**Issue by knightq** [antulik/pagelet_rails#2]
Hi,

following the doc I'm getting a: 

    ~/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/pagelet_rails-0.1.4/lib/pagelet_rails/concerns/routes.rb:29:in `pagelet_resource': undefined method `<<' for nil:NilClass (NoMethodError)
        from ~/Development/epersonam/app/pagelets/guest_info/guest_info_controller.rb:9:in `<class:GuestInfoController>'
        from ~/Development/epersonam/app/pagelets/guest_info/guest_info_controller.rb:4:in `<module:GuestInfo>'
        from ~/Development/epersonam/app/pagelets/guest_info/guest_info_controller.rb:3:in `<module:Pagelets>'
        from ~/Development/epersonam/app/pagelets/guest_info/guest_info_controller.rb:2:in `<top (required)>'
        from ~/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-4.2.6/lib/active_support/dependencies.rb:457:in `load'
on Rails boot.

Here is my controller (under) `app/pagelets/guest_info`
```ruby
# app/pagelets/guest_info/guest_info_controller.rb
module Pagelets
  module GuestInfo
    class GuestInfoController < Pagelets::BaseController
      # add pagelets_guest_info_path route
      # which gives "/pagelets/guest_info" url route
      pagelet_resource only: [:show]

      def show
      end
    end
  end
end
```

I've also tried with:
```ruby
# app/pagelets/guest_info/guest_info_controller.rb
module Pagelets
  module GuestInfo
    class GuestInfoController < ApplicationController
      include PageletRails::Concerns::Controller

      # add pagelets_guest_info_path route
      # which gives "/pagelets/guest_info" url route
      pagelet_resource only: [:show]

      def show
      end
    end
  end
end
```

but I get the same error.

**Ruby**: 2.3.1
**RoR**: 4.2.6
**pagelet_rails**: 0.1.4

**Comment by knightq** [comment]
I've solved by looking at the demo project and rewriting the controller in the following way:
```ruby
# app/pagelets/guest_info/guest_info_controller.rb
class GuestInfo::GuestInfoController < ::PageletController
  # add pagelets_guest_info_path route
  # which gives "/pagelets/guest_info" url route
  pagelet_resource only: [:show]

  pagelet_options placeholder: { height: 131 }

  def show
  end
end
```

but please, update the doc :-)

**Comment by antulik** [comment]
@knightq there was a small bug, fixed in 0.1.5. 

Also you don't need top `Pagelets` module as Rails ignores the first folder name under `/app`.
